app/code/AKisilenko/ModuleLesson9/view/frontend/templates/widget/homepagebanner.phtml

```diff
@@ -0,0 +1,32 @@
+
+
+<style>
+    .button-homepage {
+        background: <?php echo $block->getData('button_color') ?>;
+        padding: 30px;
+        border-radius: 5px;
+    }
+</style>
+
+    <?php if ($block->getButtonTitle()): ?>
+        <h2 class='custom-widget-title'>
+            <?php echo $block->getButtonTitle(); ?>
+        </h2>
+    <?php endif; ?>
+
+    <?php if ($block->getBannerContent()) : ?>
+        <p class='custom-widget-content'>
+            <?= $block->getBannerContent(); ?>
+        </p>
+    <?php endif; ?>
+<div class="home-banner">
+        <?php echo $block->getLayout()
```

Comment:
Please create the method in your block AKisilenko\ModuleLesson9\Block\Widget\Programmatically  for CMS blocks rendering and call it here